functions-and-operators/tidb-functions.md

```diff
@@ -5,24 +5,22 @@ summary: Learn about the usage of TiDB specific functions.
 
 # TiDB Specific Functions
 
-This document introduces the functions that are specific to TiDB.
+The following functions are TiDB extensions, and are not present in MySQL:
 
-## TIDB_BOUNDED_STALENESS
+| Function name | Feature description |
+| :-------------- | :------------------------------------- |
+| `TIDB_BOUNDED_STALENESS()` | The `TIDB_BOUNDED_STALENESS` function is an internal function of TiDB. |
```

Comment:
I don't think this sentence is very meaningful. See the function description in https://github.com/pingcap/docs/blob/master/as-of-timestamp.md#syntax : If you want to specify a time range, you can use the TIDB_BOUNDED_STALENESS() function in the clause. When this function is used, TiDB selects a suitable timestamp within the specified time range.

Reply:
I copied it from the previous text.

I can change it to "TIDB_BOUNDED_STALENESS() instructs TiDB to read the data as new as possible within the time range. See also: [Read Historical Data Using the `AS OF TIMESTAMP` Clause](/as-of-timestamp.md)"

Reply:
LGTM

Reply:
OK changed.